pep-0696.rst

```diff
@@ -206,20 +206,26 @@ default to the type of ``start`` and step default to ``int | None``.
    class Foo(Generic[DefaultStrT, T2]):
        def __init__(self, a: DefaultStrT, b: T2) -> None: ...
 
-   reveal_type(Foo(1, ""))       # type is Foo[int, str]
-   reveal_type(Foo[int](1, ""))  # type is Foo[int, str]  # example of partial specialisation
-   Foo[int]("", 1)               # Invalid: Foo[str, int] cannot be assigned to self: Foo[int, int] in Foo.__init__
+   reveal_type(Foo(1, ""))  # type is Foo[int, str]
+   Foo[int](1, "")          # Invalid: Foo[int, str] cannot be assigned to self: Foo[int, int] in Foo.__init__
+   Foo[int]("", 1)          # Invalid: Foo[str, int] cannot be assigned to self: Foo[int, int] in Foo.__init__
 
 When using a ``TypeVarLike`` as the default to another ``TypeVarLike``.
 Where ``T1`` is the default for ``T2`` the following rules apply.
 
+Note
+''''
+Currently ``TypeVarTuple``\s as defaults are not supported due to
```

Comment:
I recommend removing the word "Currently" because it's confusing. A PEP is meant to be a proposed design, so it should be unequivocal about what's supported and what's not. Of course, future changes can always change or extend a design. It may be worth adding a note in the "Rejected Ideas" section that explains why this was rejected. You can state there that this could be added in the future.